### PR TITLE
Inconsistencies in dark and default mode removed, Solves #531

### PR DESCRIPTION
**Pull request by shikhar-scs**
As suggested by @Natim and marked out by @SoftVision-CarmenFat at #531, the inconsistencies have  been removed. Here's a preview of the same (its a long gif file, please be patient while it loads)

fornotes

Please preview and suggest anymore changes to be made.

### CODE REVIEW

File: src/sidebar/styles-dark.css [hunk 1]
```diff
@@ -25,10 +25,11 @@ body {
 /* material-design-lite (MDL) menu */
 #context-menu-button {
   background-image: url('dark-svg/three-dots-dark.svg');
+  background-color: #323744;
 }
 
 #footer-buttons.warning #context-menu-button {
-  background-color: rgba(0, 0, 0, 0.5) !important;
+  /*background-color: rgba(0, 0, 0, 0.5) !important;*/
```
Comment (Natim) [inline src/sidebar/styles-dark.css]: How does it works with warnings? Could you show us with a screenshot? (You can try to login to get a warning footbar)

Comment (shikhar-scs) [reply src/sidebar/styles-dark.css]: how can I produce a warning ? Any ideas

Comment (shikhar-scs) [reply src/sidebar/styles-dark.css]: Something like this maybe ?

fornotessss

Comment (Natim) [reply src/sidebar/styles-dark.css]: I guess it shouldn't be dark on the dark theme but the same as the light theme what do you think?

Comment (shikhar-scs) [reply src/sidebar/styles-dark.css]: Yeah you're right
I'll make changes and push that rightaway

Comment (shikhar-scs) [reply src/sidebar/styles-dark.css]: yoyoyoyo

**Comment by Natim** [comment]
Thanks @shikhar-scs

**Comment by shikhar-scs** [comment]
Welcome @Natim :smile:

**Pull request merged**